elements/lisk-validator/src/formats.ts

```diff
@@ -125,3 +127,41 @@ const camelCaseRegex = /^[a-z]+((\d)|([A-Z0-9][a-zA-Z0-9]+))*([a-z0-9A-Z])?$/;
 
 export const camelCase = (data: string): boolean =>
 	camelCaseRegex.exec(data) !== null;
+
+export const version = (data: string): boolean =>
+	/^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})(-(alpha|beta|rc)\.[0-9]{1,3}(\.[0-9]{1,3})?)?$/.test(
+		data,
+	);
+
+export const protocolVersion = (data: string): boolean =>
+	/^(\d|[1-9]\d{1,2})\.(\d|[1-9]\d{1,2})$/.test(data);
```

Comment:
I think you want to it to be 2 digit protocol versions as `00.00`, but this expression will match 3 digits in each group.